### PR TITLE
developer environment setup and server setup

### PR DESCRIPTION
**Pull request by jsun98**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,26 +4,36 @@
...
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "export NODE_ENV=development && babel-node --presets es2015 server/index.js"
+    "start-server": "babel-node server/server.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/eric-zhu-uw/request.git"
   },
-  "author": "",
-  "license": "ISC",
```
Comment (Tilmanator) [inline package.json]: Where is my license to code?

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+global.navigator = global.navigator || {};
+global.navigator.userAgent = global.navigator.userAgent || 'all';
+
+app.use(express.static(path.resolve(__dirname, 'public')));
+app.use(cookieParser());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+if (process.env.node_env) {
```
Comment (eric-zhu-uw) [inline server/server.js]: I'm pretty sure it's normally env.NODE_ENV

**Comment by AkshayPall** [comment]
Merge for merge

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+// eslint-disable-next-line no-unused-vars
+app.use((err, req, res, next) => {
+  console.error(pe.render(err));
+  res.status(err.status || 500);
+  res.send(pe.render(err));
+});
+
+app.listen(config.port, () => {
+  console.info(`The server is running at http://localhost:${config.port}/`);
```
Comment (eric-zhu-uw) [inline server/server.js]: should probably log node_env here

**Pull request merged**